jbmc/regression/jbmc/NondetEnumArg/NondetEnumArg.java

```diff
@@ -0,0 +1,46 @@
+public class NondetEnumArg {
+
+    public static void canChooseSomeConstant(Color c) {
+        if (c == null)
+            return;
+        assert c != null;
+        boolean isRed = c.name().startsWith("RED") && c.name().length() == 3
```

Comment:
May we have tests with `switch` already?

Reply:
We have one tests for `switch` on an enum type. It uses `--cover location` and pretty much just checks that all possible ordinals (except the last one) can be generated for a switch statement.

Reply:
@peterschrammel Do you think the `switch` test and the new `NondetEnumArg` test should be combined?